scripts/fix_biorxiv_xml.js

```diff
@@ -13,7 +13,15 @@ function fixXml(path) {
     fileContent = fileContent.replace(new RegExp('</table-wrap>', 'g'), '</fig>'); // # table-wrap -> fig
 
     // replace more complex patterns
-    fileContent = fileContent.replace(new RegExp('<label>(.*)<\/label>[\r\n]*<title>', 'mg'), '<title>'); // <label>1</label><title> -> <title><label>1</label>
+
+    // <label>1</label><title> -> <title><label>1</label>
+    fileContent = fileContent.replace(new RegExp('<label>(.*)<\/label>[\r\n]*<title>', 'mg'), '<title>');
```

Comment:
Not for now but these tricky little replaces should be under tests otherwise they are really scary to change. I will raise a ticket to capture that tech debt.

Reply:
Yup, totally. I would imagine this moving into an import mechanism (that doesn't yet exist) and coming under test then. For now, this was me trying to at least capture edits we were being asked to make